### ISSUE TITLE
concurrentScavenge system tests failing on OSX Unknown option: concurrentScavenge

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#3830]
https://ci.eclipse.org/openj9/job/Test-sanity.system-JDK11-osx_x86-64_cmprssptrs/1
```
CLT stderr JVMJ9GC040E Unknown option: concurrentScavenge
CLT stderr JVMJ9VM015W Initialization error for library j9gc29(2): Failed to initialize; unable to parse command line
```

**Comment by pshipton** [comment]
@amicic @0dvictor @llxia 
We either need to add concurrentScavenge support to OSX or exclude the testing.

**Comment by smlambert** [comment]
I would prefer directly excluding it on OSX, than using the mode string, since we should be moving away from using modes and prefer the use of the full string of options.  

I understand that what is blocking us from whole-heartedly removing references to modes is that we carried over from the old/proprietary test infrastructure the mechanism that checks if a particular mode is valid on a particular platform/spec.  I'll add to the 2019 test plan an item to rethink how this can be accomplished in the new world (relates also to the recent modes service discussion in https://github.com/eclipse/openj9/pull/3801).

**Comment by smlambert** [comment]
https://github.com/AdoptOpenJDK/openjdk-tests/pull/741

**Comment by smlambert** [comment]
yes, CS tests skipped on osx, sanity.system target is now running 'green'
https://ci.eclipse.org/openj9/view/Test/job/Test-sanity.system-JDK11-osx_x86-64_cmprssptrs/15/tapResults/
